composer.json

```diff
@@ -59,8 +59,9 @@
     "doctrine/dbal": "^2.12",
     "doctrine/annotations": "^1.13",
     "laminas/laminas-servicemanager": "~2.5.0",
-    "league/flysystem": "~1.0",
-    "league/flysystem-memory": "~1.0",
+    "league/flysystem": "^2.0",
+    "league/flysystem-memory": "^2.0",
+    "league/flysystem-google-cloud-storage": "^2.0",
```

Comment:
could you please validate does public contract of this library affect by those chages?

Reply:
It shouldn't be a problem because I apply all changes here https://github.com/oat-sa/generis/blob/78bbd6aa558b4db85715d0efa8cea6c597a0f58b/common/oatbox/filesystem/wrapper/GoogleStorageWrapper.php

and into trait which write all changes